Tests/FixtureIntegrationTest.php

```diff
@@ -29,6 +29,7 @@
 use function sprintf;
 use function sys_get_temp_dir;
 
+/** @group legacy */
```

Comment:
This is probably not the way to go, but it was a nice way for me to toggle deprecation triggers with phpunit easily. Will figure out the correct approach here

Reply:
Without changes to the configuration, the extension will always call the deprecated method in ODM. This needs changing in `DoctrineMongoDBExtension::loadDocumentManager`, where we add the call to `setMetadataCacheImpl`:
https://github.com/doctrine/DoctrineMongoDBBundle/blob/3f6449eb05ff5395f90d834b3ff87be3b33d62d7/DependencyInjection/DoctrineMongoDBExtension.php#L208

Changing this to `setMetadataCache` (after bumping the ODM dependency to `^2.3`) will fix this.

Reply:
Yeah removed this again, should be correct now.